src/UI/WB.UI.Headquarters.Core/Startup.cs

```diff
@@ -228,6 +226,14 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddUnderConstruction();
 
+            services.Configure<ForwardedHeadersOptions>(options =>
+            {
+                options.ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto;                    
```

Comment:
I can't remember, but there were some reason why I haven't used this default from MS documentation example. Maybe this is were Traefik related or something else. Can You check that samples from `docker-compose` repo is still working with this change?

Reply:
Revert that flag back. No need to change it, was fixing another issue and changed to recommended.